### ISSUE TITLE
When reduced transparency enabled circle buttons are not visible properly.

### ISSUE THREAD

**Issue by madnik** [carsonperrotti/CNPGridMenu#14]
When I enabled reduce transparency under Settings->General->Accessibility->Increased Contrast, the circle buttons almost not visible.
